### DISCUSSION TITLE
What would I need to change for this to work for S1 plus?  I changed to non touchscreen

### DISCUSSION THREAD

**Discussion by mattfox27** [mriscoc/Ender3V2S1#1035] [General]
I installed a knob screen and prof firmware on my s1 plus, I then changed my bed size to 290x290 but whenever I goto level or tram it wont go all the way to the right.  Do I have to change something in firmware and recompile?

**Comment by mriscoc** [comment]
Follow the guide in the wiki https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
